buildLibraries.sh

```diff
@@ -41,6 +50,12 @@ if [ $BOINC = true ] || [ $ALL = true ]; then
     cd ..
 fi
 
+if [ $ZLIB = true ] || [ $ALL = true ]; then
+    cd lib
+    ./buildZlib.sh
```

Comment:
I think this must be `source'd` to get `$ZLIB_BASE`? Also this probably needs to `export`ed to propagate into the other sub build scripts.

Reply:
For HDF5 the path seems hardcoded...

Reply:
Fixed with source / export